feat: added log file

libialany opened the pull request:
add log file feature

libialany committed:
feat: added log file

LuisMitaHL commented on debian-reset.sh:
```diff
@@ -8,11 +8,12 @@ echo
 echo "Docs: https://github.com/LuisMitaHL/debian-reset"
 echo
 
+
```
Hay un espacio adicional, serìa bueno quitarlo del commit

libialany replied:
i deleted the new line

LuisMitaHL commented on debian-reset.sh:
```diff
@@ -8,11 +8,12 @@ echo
   GITHUB_URL="https://raw.githubusercontent.com/LuisMitaHL/debian-reset/main/debian-reset.sh"
   echo "Actualizando script..."
-  curl -L -o "$0.new" "$GITHUB_URL"
+  curl -L -o "$0.new" "$GITHUB_URL" 
```
Hay un espacio adicional, serìa bueno quitarlo del commit

libialany replied:
i remove empty spaces

LuisMitaHL commented on debian-reset.sh:
```diff
@@ -25,7 +26,8 @@ fi
 # TODO: revisar si se pudo elevar
 if [[ $EUID -ne 0 ]]; then
-  echo "No tenemos permisos de superusuario. Intentando elevar..."
+  echo "No tenemos permisos de superusuario. Intentando elevar..." >> $HOME/debian-reset.log
```
Los mensajes de echo son más valiosos en la terminal para que la gente lo vea, pero >> hace que no se muestren
Podría usarse "tee" o no usarse nada

libialany replied:
my mistake

LuisMitaHL reviewed (changes requested):
Capaz el comando >> deba usarse en los comandos más verbosos (apt por ejemplo)

libialany committed:
Update debian-reset.sh

- fix spaces
- remove unused redirection
- add  cmds for nginx uninstalling proces
- add msg redirecting nginx port

libialany committed:
Update debian-reset.sh

LuisMitaHL commented:
Please merge your changes to one commit

LuisMitaHL commented:
I don't see any changes related to adding a log file. Please open a new PR for the new changes.

libialany committed:
Merge branch 'LuisMitaHL:main' into feat/logs
